src/Security/Security/Az.Security.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.9'
+ModuleVersion = '0.8.0'
```

Comment:
please do not modify the module version here, Azure-PowerShell team will bump it during each release

Reply:
Done, thanks for your guidance.

Reply:
please move this version back to 0.7.9